addons/account_peppol/models/account_edi_proxy_user.py

```diff
@@ -202,6 +202,9 @@ def _peppol_get_message_status(self):
 
                 move = message_uuids[uuid]
                 if content.get('error'):
+                    if content['error'].get('code') == 702:
```

Comment:
Better to explicit the magic number with a comment.

```suggestion
                    if content['error'].get('code') == 702  # "Peppol request not ready"
```

Reply:
thanks, I forget sometimes that people might not know what is going on exactly.
That being said, I reworded your suggestion, as it's not the IAP checking the move, but the client side